Rakefile

```diff
@@ -49,12 +63,15 @@ task :generate_policy_list do
       file_list<<{
         "name": @name,
         "file_name": file,
-        "version": @version,
+        "version": version,
         "change_log": change_log,
         "description": @description,
         "category": @category,
         "severity": @severity,
-        "readme": readme
+        "readme": readme,
+        "provider":provider,
+        "service":service,
+        "policy_set":policy_set
```

Comment:
This is more of a style comment than anything else, but usually there is a space after the `:` and having a trailing `,` here would prevent the extra diff line above from happening in the future.